packages/lib-react-components/src/AnimatedNumber/AnimatedNumber.js

```diff
@@ -4,58 +4,67 @@ import { interpolate } from '@visx/vendor/d3-interpolate'
 import { select } from 'd3'
 import { useEffect, useRef, useState } from 'react'
 
-const initialValue = 0
-
 let prefersReducedMotion
 const isBrowser = typeof window !== 'undefined'
 if (isBrowser) {
   prefersReducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)').matches
 }
 
+function formatValue(num) {
+  return format(',d')(num)
+}
 function AnimatedNumber({ duration = 1000, value }) {
   const numRef = useRef(null)
+  const initialValueRef = useRef(0)
+  const initialValue = initialValueRef.current
   const [animated, setAnimated] = useState(false)
 
-  function animateValue() {
-    select(numRef.current)
-      .data([value])
-      .transition()
-      .duration(duration)
-      .textTween(() => {
-        const interpolator = interpolate(initialValue, value)
-        return t => {
-          const interpolatedValue = interpolator(t)
-          if (interpolatedValue === value) setAnimated(true) // animation complete!
-          const niceValue = formatValue(interpolatedValue)
-          return niceValue
-        }
-      })
-  }
-
-  function lessAnimation() {
-    select(numRef.current)
-    .data([value])
-    .transition()
-    .duration(0)
-    .textTween(() => {
-      return () => {
-        setAnimated(true) // animation complete!
-        return formatValue(value)
-      }
-    })
-  }
-
-  function formatValue(num) {
-    return format(',d')(num)
-  }
-
   useEffect(() => {
     // If we already animated the number once, don't observe intersection
     // This could happen if the value prop updates, but page did not refresh
     if (animated) return
 
     const numElement = numRef.current
 
+    function animateValue() {
+      if (value === initialValue) {
+        return
+      }
```

Comment:
This should fix #6190 by holding off the animation until `value`  has loaded from Zooniverse APIs.